app-text/sigil/sigil-0.8.901_pre.ebuild

```diff
@@ -23,10 +23,17 @@ KEYWORDS="~amd64 ~x86"
 IUSE=""
 
 RDEPEND="
-	dev-lang/python:3.4
-	>=dev-libs/boost-1.49[threads]
-	>=dev-libs/libpcre-8.31[pcre16]
-	>=dev-libs/xerces-c-3.1.1[icu]
+	dev-libs/boost[threads]
+	dev-libs/libpcre[pcre16]
+	dev-libs/xerces-c[icu]
+	dev-python/chardet
```

Comment:
All dependencies on Python packages need `[${PYTHON_USEDEP}]` to request matching Python versions.